LiteLoader/Header/MC/Types.hpp

```diff
@@ -713,6 +713,26 @@ enum class ContainerType : char {
     SMITHING_TABLE         = 33,
 };
 
+enum class GameType : int {
```

Comment:
enum GaneType shoule be `enum` instead of `enum class` 
ref: https://github.com/LiteLDev/LiteLoaderBDS/blob/e601f387cc6c6c58fc4382d90e74fcc16af6073c/LiteLoader/Header/MC/GameTypeConv.hpp#L29

Reply:
it's better to move this enum to Player.hpp
also GameMode::Spectator is missing

Reply:
> it's better to move this enum to Player.hpp also GameMode::Spectator is missing

**Fixed!**

Reply:
> it's better to move this enum to Player.hpp

this isn't working. check [this](https://github.com/LiteLDev/LiteLoaderBDS/runs/8168820512?check_suite_focus=true#step:10:294
)